cspm/rn/prisma-cloud-compute-release-information/look-ahead-planned-updates-prisma-cloud-compute.adoc

```diff
@@ -5,18 +5,60 @@ Review any deprecation notices and new features planned in the next Prisma Cloud
 
 See xref:prisma-cloud-compute-release-information.adoc#id79d9af81-3080-471d-9cd1-afe25c775be3[Prisma Cloud Compute Release Information] for the latest features of the host, container, and serverless capabilities that are available on the *Compute* tab on Prisma Cloud.
 
-The latest release is 31.01.123 (September 10, 2023).
+//The latest release is 31.01.123 (September 10, 2023).
 
-//The details and functionality listed below are a preview of what is planned in the next Compute update planned for September 10, 2023; the changes listed herein and the actual release date, are subject to change.
+The details and functionality listed below are a preview of what is planned in the next Compute update planned for October 8, 2023; the changes listed herein and the actual release date, are subject to change.
 
 //=== Defender Upgrade
 
 // === Changes in Existing Behavior
 
 // === Deprecation Notices
 
-// === End of Support Notices 
+=== End of Support Notices
 
-//=== Addressed Issues
+[cols="50%a,50%a"]
+|===
+
+//CWP-51558
+|Ubuntu 18.04 LTS EOL
+|Ubuntu 18.04 reached End of Standard Support May 31, 2023. If you still have devices in your fleet using Ubuntu 18, follow https://ubuntu.com/blog/ubuntu-18-04-eol-for-devices[Canonical's guidance to keep your devices up and running]. Starting with Newton Update 2, Prisma Cloud Compute is no longer tested on devices running Ubuntu 18 and no support for those devices is available.
+
+|===
+
+=== Addressed Issues
+
+[cols="50%a,50%a"]
+|===
+
+|-
+//CWP-51616
+|Fixed the connection issue whereby Prisma Cloud failed to retrieve image details for Docker registries and displayed HTTP 429 errors when scanning more than one Docker registry at a time.
+
+|-
+//CWP-51942
+|Fixed the API rate limit for the following GET requests:
+
+/api/v31.01/containers
+/api/v31.01/hosts
+/api/v31.01/images
+
+With this fix, the API rate limit is set back to 30 requests in 30 seconds.
+
+|===
+
+=== Enhancements
+
+The following enhancements are planned; the details will be available at release:
+
+//CWP-44646
+* Support for Scanning Images based on Wolfi Linux
```

Comment:
@Pubs-MV Do we want the description of the blurbs or just the headings like we did in [Newton update 1 LookAhead](https://github.com/PaloAltoNetworks/prisma-cloud-docs/pull/1501/files0?

Reply:
Adding just the headings as the blurb is incomplete/missing and we do not know which are customer asks.